misc/SW_WA_2021_2024/guilderton_to_geraldton/inverts/README.md

```diff
@@ -0,0 +1,12 @@
+This directory contains a script to create x/y/z data representing the bottoms of channels.
+
+These can be 'burned' into the elevation grid in SWALS to ensure that flow paths
+are represented, even if sometimes the model is too coarse to reliably
+capture the the minima.
+
+The code is very similar to that in ../breakwalls
```

Comment:
Maybe add something like "Create the breakwalls as line shapefiles in subdirectories -- the script will operate on any shapefiles matching `*/*.shp`"